conans/client/rest/conan_requester.py

```diff
@@ -17,6 +18,15 @@
 # TODO: Fix this security warning
 logging.captureWarnings(True)
 
+class SSLContextAdapter(HTTPAdapter):
+    def init_poolmanager(self, *args, **kwargs):
+        context = create_urllib3_context()
+        kwargs['ssl_context'] = context
+        try:
+            context.load_default_certs()
+        except AttributeError:
+            pass # load_default_certs is not available on Mac OS X
```

Comment:
This works for some versions. It not only depends on the [`urllib3` version](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util) (the one bundled in `requests` or taken from the system), there should be something else in the `requests` sources. IMO, I think it is ok (needed) to maintain this `try/except`. 👍